netstandard/ref/netstandard.csproj

```diff
@@ -9,15 +9,17 @@
     <IsCoreAssembly>true</IsCoreAssembly>
     <!-- disable obsolete warnings/errors -->
     <NoWarn>$(NoWarn);0618;0619;0809</NoWarn>
-
     <DefaultBuildAllTarget>Build</DefaultBuildAllTarget>
     <BuildAllProjects>true</BuildAllProjects>
+    <SignType>oss</SignType>
```

Comment:
https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/sign.targets#L6 That tells me that it shouldn't trying to StrongName sign it for the Open key.